src/main/java/domain/Position.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import java.util.Objects;
+
+/**
+ * 위치 구현체
+ */
+public class Position implements Comparable<Position> {
+    private int value;
```

Comment:
```suggestion
    private final int value;
```
value object 👍 불변한 속성을 부여해주면 좋을 것 같아요 :)

Reply:
@jinyoungchoi95 진영님, 안녕하세요! 리뷰 남겨주신것 다시 한번 확인해 보고 있는데요~ 혹시 value에 final을 부여하게 되면 Position이 변경될 때마다 새로운 객체를 생성해야 하는 방법 외에는 떠오르지 않는데, 다른 방법이 있나요?